### DISCUSSION TITLE
Attending the marketing committe meeting

### DISCUSSION THREAD

**Discussion by fbouteruche** [dotnet-foundation/Home#81] [Marketing Committee]
Hi there. I would love to attend some of the committee meetings to see how it goes and decide if it is the good fit for me. At an individual level, I would like to increase my contribution to the foundation.

**Comment by isaacrlevin** [comment]
hey there! The easiest way would be to get a personal email address added to marketing distribution list. Can you please send an email to isaac.levin@dotnetfoundation.org and I will get that setup for you.
